app_dart/lib/src/request_handlers/github_webhook.dart

```diff
@@ -181,6 +183,29 @@ class GithubWebhook extends RequestHandler<Body> {
     await scheduler.triggerPresubmitTargets(pullRequest: pr);
   }
 
+  /// Release tooling generates cherrypick pull requests that should be granted an approval.
+  Future<void> _tryReleaseApproval(
```

Comment:
Part of the release process requires that two people get involved in release PRs this change will allow unilateral changes. @Hixie thoughts?

Reply:
i don't mind this for the cherrypick step, so long as the actual release (once the cherrypicks are known) requires two people to sign off.

Reply:
I'm less concerned about getting reviews on beta and stable releases as those are reviewed weekly. This is to unblock the g3 roller where we're CPing changes into the internal release branches.

My long term plan here is there would be no human accounts that would get sign off, but instead automated bot accounts creating CP PRs would get approved.